gix-blame/src/file/function.rs

```diff
@@ -400,12 +424,19 @@ fn find_path_entry_in_commit(
     file_path: &BStr,
     buf: &mut Vec<u8>,
     buf2: &mut Vec<u8>,
+    stats: &mut Statistics,
 ) -> Option<gix_object::tree::Entry> {
     let commit_id = odb.find_commit(commit, buf).unwrap().tree();
     let tree_iter = odb.find_tree_iter(&commit_id, buf).unwrap();
+    stats.commits_to_tree += 1;
+    stats.trees_decoded += 1;
 
     tree_iter
-        .lookup_entry(odb, buf2, file_path.split(|b| *b == b'/'))
+        .lookup_entry(
+            odb,
+            buf2,
+            file_path.split(|b| *b == b'/').inspect(|_| stats.trees_decoded += 1),
```

Comment:
This seems to be an unusual location for `+= 1`.

Reply:
Yes, definitely, but it's simple. It's also over-counting the last component, which I realise now, can actually be fixed. That will be done in a moment.